sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -52,22 +53,33 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     this.logger.log("[Created]");
   }
 
-  async createAssociation(
-    institution: Institution,
-    user: User,
-    type: InstitutionUserType = InstitutionUserType.user,
-    role?: InstitutionUserRole,
-  ): Promise<InstitutionUser> {
+  async createAssociation({
+    institution,
+    type = InstitutionUserType.user,
+    user,
+    location,
+    role,
+    guid,
+  }: {
+    institution: Institution;
```

Comment:
Can you please consider creating a type for this?

Reply:
This is inline param type, do we really need to create any type for this?

Reply:
This is inline type for method param. Do we really need explicit type def for this?